### PR TITLE
sep dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-september-2024.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+Work on the Python Rule Engine plugin is progressing well. An issue involving the GIL and concurrent access was resolved in the lead up to 4.3.3. Aside from that, we're working hard to add support for virtual environments. If done correctly, users will be able to use a different set of modules than those provided by their system's package manager. You can follow that effort at [https://github.com/irods/irods_rule_engine_plugin_python/pull/224](https://github.com/irods/irods_rule_engine_plugin_python/pull/224).
+
+For the Automated Ingest capability, a new Docker Compose project was added for demo purposes. Work to provide a clean separation between filesystem scanner tasks and S3 scanner tasks is in progress. We expect that to land very soon. The big news is we've successfully demonstrated a new Celery task which recursively removes data in parallel. Early performance testing has shown a 55% improvement over `irm -rf`.
```
Comment (trel) [inline content/posts/irods-development-update-september-2024.md:20]:
```suggestion
For the Automated Ingest capability, a new Docker Compose project was added for demonstration purposes. Work to provide a clean separation between filesystem scanner tasks and S3 scanner tasks is in progress. We expect that to land very soon. The big news is we've successfully demonstrated a new Celery task which recursively removes data in parallel. Early performance testing has shown a 55% improvement over `irm -rf`.
```

File: content/posts/irods-development-update-september-2024.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        - released 4.3.3.0
+
+    - [Logical Quotas](https://github.com/irods/irods_rule_engine_plugin_logical_quotas)
+        - released 4.3.3.0
+
+    - [Metadata Guard](https://github.com/irods/irods_rule_engine_plugin_metadata_guard)
+        - released 4.3.3.0
+
+    - [CURL microservices](https://github.com/irods/irods_microservice_plugins_curl)
+        - released 4.3.3.0
```
Comment (trel) [inline content/posts/irods-development-update-september-2024.md:130]: note... many of these might move to 'background' for next month.

Comment (korydraughn) [reply content/posts/irods-development-update-september-2024.md:130]: True. It's not clear to me why that matters?

File: content/posts/irods-development-update-september-2024.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+
+The PAM Interactive Authentication plugin is now available for iRODS 4.3.3. See [https://irods.org/2024/08/initial-release-of-the-irods-pam-interactive-auth-plugin/](https://irods.org/2024/08/initial-release-of-the-irods-pam-interactive-auth-plugin/) for details.
+
+Work on the Python Rule Engine plugin is progressing well. An issue involving the GIL and concurrent access was resolved in the lead up to 4.3.3. Aside from that, we're working hard to add support for virtual environments. If done correctly, users will be able to use a different set of modules than those provided by their system's package manager. You can follow that effort at [https://github.com/irods/irods_rule_engine_plugin_python/pull/224](https://github.com/irods/irods_rule_engine_plugin_python/pull/224).
```
[the remark this answers is not in the document]
Comment (korydraughn) [reply content/posts/irods-development-update-september-2024.md]: Done.

File: content/posts/irods-development-update-september-2024.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-september-2024.md]: Done.

File: content/posts/irods-development-update-september-2024.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+
+For the Automated Ingest capability, a new Docker Compose project was added for demo purposes. Work to provide a clean separation between filesystem scanner tasks and S3 scanner tasks is in progress. We expect that to land very soon. The big news is we've successfully demonstrated a new Celery task which recursively removes data in parallel. Early performance testing has shown a 55% improvement over `irm -rf`.
+
+Version 0.4.0 of the iRODS HTTP API is now available. You can view the full list of closed issues at [https://github.com/irods/irods_client_http_api/milestone/5?closed=1](https://github.com/irods/irods_client_http_api/milestone/5?closed=1).
+
+The iRODS S3 API now supports the **AbortMultipartUpload** operation. The performance improvements for multipart upload are now complete. All that's left to do is to polish the changes. We expect the updates to be merged into upstream within a few weeks.
+
+This is a big one, Jargon 4.3.4.0-RELEASE is now available for download! To learn more, see [https://github.com/DICE-UNC/jargon/releases/tag/4.3.4.0-RELEASE](https://github.com/DICE-UNC/jargon/releases/tag/4.3.4.0-RELEASE).
```
[the remark this answers is not in the document]
Comment (korydraughn) [reply content/posts/irods-development-update-september-2024.md]: Went with `-`.

Done.

**Comment by trel** [comment]
squash em.

**Comment by korydraughn** [comment]
Squashed.

Let me know if you'd like to discuss the plugins moving to the background activity section.

**Pull request merged**